shiksha-website/shiksha-frontend/src/app/auth/sign-in/sign-in.component.html

```
@@ -34,41 +34,43 @@ <h1 class="md:text-3xl text-xl font-semibold header-text text-content">Welcome</
 							<p *ngIf="numberErrorMsg" class="text-error text-left">{{numberErrorMsg}}</p>
 						</div>
 
 						<div class="flex items-center gap-1 mb-4">
-							<input type="checkbox" [(ngModel)]="rememberMe" name="rememberMe">
-							<span class="text-sm text-[#424955]">Remember Me</span>
+							<input type="checkbox" id="rememberMe" [(ngModel)]="rememberMe" name="rememberMe"
+								aria-label="Remember Me">
+							<label for="rememberMe" class="text-sm text-[#424955]">Remember Me</label>
 						</div>
 
 						<button class="btn-primary w-full shadow-lg" (click)="onVerifyPhoneNumber()">Continue <img
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)